src/liveEditor/components/pseudoEditableField.tsx

```diff
@@ -0,0 +1,41 @@
+import { getStyleOfAnElement } from "../generators/generatePseudoEditableField";
+
+interface PseudoEditableFieldProps {
+    editableElement: HTMLElement;
+    config: { textContent: string };
+}
+
+function PseudoEditableFieldComponent(props: PseudoEditableFieldProps) {
+    const styles = convertToCamelCase(
+        getStyleOfAnElement(props.editableElement)
+    );
+    const { top, left } = props.editableElement.getBoundingClientRect();
+
+    styles.position = "absolute";
+    styles.top = `${top}px`;
+    styles.left = `${left}px`;
+
+    return (
+        <div
+            className="visual-editor__pseudo-editable-element"
+            data-testid="visual-editor__pseudo-editable-element"
+            style={styles}
+        >
+            {props.config.textContent}
+        </div>
+    );
+}
+
+function convertToCamelCase(styles: { [key: string]: string }): {
+    [key: string]: string;
+} {
+    return Object.keys(styles).reduce((acc, key) => {
+        const camelCasedKey = key.replace(/-([a-z])/g, (g) =>
+            g[1].toUpperCase()
+        ) as string;
+        acc[camelCasedKey] = styles[key];
+        return acc;
+    }, {} as { [key: string]: string });
+}
```

Comment:
If not, let's use some package that does this for us.

Reply:
No it wasn't present earlier. The closest I could find was this package called `camelcase-css`. But that only does that transformation one at a time. Should I use that package in this function ? Because we need to convert the entire stylesheet into a CSS style object.

Reply:
Keep the current one. For the acual conversion, you can use just-camel-case.